### ISSUE TITLE
Emails - problem importing emails with special chars it the subject

### ISSUE THREAD

**Issue by jneto81** [oroinc/platform#870]
vendor/zendframework/zend-mail/src/Header/GenericHeader.php
line 33

`        $decodedLine = iconv_mime_decode($headerLine, ICONV_MIME_DECODE_CONTINUE_ON_ERROR, 'UTF-8');
`

Work well with php 7.1.21

Error with php 7.1.22 ( they made changes in iconv_mime_decode )

My temporary fix was changing to 

`        $decodedLine = iconv_mime_decode($headerLine, ICONV_MIME_DECODE_STRICT, 'UTF-8');
`

Using ORO version 3.1.0-beta

**Comment by jneto81** [comment]
Hello, more problems again with other emails.
 
Added the imap_utf8 to fix it for me. But i need a clean solution, without changing
vendor/zendframework/zend-mail/src/Header/GenericHeader.php


    ```
    $decodedLine = @iconv_mime_decode($headerLine, ICONV_MIME_DECODE_STRICT , 'UTF-8');

       if ( $decodedLine === false  ) {
             $decodedLine = imap_utf8(  $headerLine );        
        }
        
```

**Comment by jneto81** [comment]
I'm closing this issue because it seems fixed with ORO 3.1.9.
